packages/expo-modules-test-core/ios/ExpoModulesTestCore.podspec

```diff
@@ -19,6 +19,6 @@ Pod::Spec.new do |s|
   s.source_files   = '**/*.{h,m,mm,swift}'
 
   s.dependency 'ExpoModulesCore'
-  s.dependency 'Quick'
-  s.dependency 'Nimble'
+  s.dependency 'Quick', '~> 5.0.0'
+  s.dependency 'Nimble', '~> 9.2.0'
```

Comment:
i've seen Nimble.podspec bumped to 9.2.1. we may better to use 9.2.1 here, too.

Reply:
For now I don't want be too strict, `9.2.1` matches the requirement of `~> 9.2.0` so that's fine I think. Also, our custom podspec uses `9.2.1` already.
Side note: `10.0.0` should be released soon (https://github.com/Quick/Nimble/discussions/979), also with my fix that would allow us to remove the custom podspec

Reply:
that makes sense to me. thanks for the explanation.